packages/internal-testing-tools/ios/Sources/DdInternalTestingImplementation.swift

```diff
@@ -12,22 +12,22 @@ import DatadogInternal
 
 @objc
 public class DdInternalTestingImplementation: NSObject {
-    private var coreProxy: DatadogCoreProxy? = nil
-
     @objc
     public func clearData(resolve:RCTPromiseResolveBlock, reject:RCTPromiseRejectBlock) -> Void {
-        self.coreProxy?.waitAndDeleteEvents(ofFeature: "rum")
-        self.coreProxy?.waitAndDeleteEvents(ofFeature: "logging")
-        self.coreProxy?.waitAndDeleteEvents(ofFeature: "tracing")
-        self.coreProxy?.waitAndDeleteEvents(ofFeature: "session-replay")
+        let coreProxy = (DatadogSDKWrapper.shared.getCoreInstance() as! DatadogCoreProxy)
```

Comment:
I saw another potential solution to create a singleton to hold the proxied core instance, but I felt there was no point in adding another singleton to the one we already had that will return the proxied core given `enable` was called before.